src/server/gdb_server.c

```diff
@@ -3036,6 +3036,19 @@ static bool gdb_handle_vcont_packet(struct connection *connection, const char *p
 
 	/* simple case, a continue packet */
 	if (parse[0] == 'c') {
+		if (target->state == TARGET_UNAVAILABLE) {
+			struct target *available_target = get_available_target_from_connection(connection);
+			if (target == available_target) {
+				/* If the only target is unavailable, then just
+				 * fake to gdb that it is now running. */
```

Comment:
I'd suggest a reword - to make the comment more accurate:

```suggestion
				/* All targets for this gdb connections are unavailable --
				 * fake to gdb that the resume succeeded and the target is now running. */
```

Reply:
Done. Thanks.

Reply:
@timsifive In the current diff, I can still see the original code. 

Can you please double-check whether your latest changes have been pushed?

Reply:
Yup. Not pushed yet. I tend to comment before I make the change, and then I needed to run tests, and then got side-tracked. But it's coming.